daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/DataProcessor.scala

```diff
@@ -295,6 +306,7 @@ class DataProcessor(
       variableMap = ssrd.originalVariables, // reset to original variables defined in schema
       validationMode =
         ValidationMode.Off, // explicitly turn off, so restored processor won't be validating
+      diagnostics = Seq.empty,
```

Comment:
Might be worth adding a comment like the other ones so we document that we are just dropping warnings.

Reply:
Updated